app/models/tickets_edit_person.rb

```diff
@@ -11,11 +11,8 @@ class TicketsEditPerson < ApplicationRecord
   has_one :ticket, as: :metadata
   has_many :tickets_edit_person_fields
 
-  def action_user_groups(action)
-    case action
-    when :update_status
-      [UserGroup.teams_committees_group_wrt]
-    end
+  def actions_allowed
+    %i[add_comment update_status]
```

Comment:
These action literals should probably be standardized into constants somewhere, to reduce the risk of typos (for example `if actions_allowed.include?(:update_stauts)` or similar) as much as possible

Reply:
As per our discussion in chat, I'll do this in a separate PR after this.